ocs_samples/basic_samples/SDS/Java/sdsjava/src/main/java/com/github/osisoft/sdsjava/Program.java

```diff
@@ -512,7 +521,8 @@ public static boolean toRun() {
             e.printStackTrace();
         } finally {
             try {
-                // Step 20
+                // Step 21
+				System.out.println();
```

Comment:
Not sure why this println() shows up out there. In the editor it was in line with the rest of the code.